test/ui-test/extensionUITest.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable no-console */
+/* eslint-disable func-names */
+import { expect } from 'chai';
+import { ActivityBar, ExtensionsViewSection, ExtensionsViewItem, VSBrowser, NotificationType } from 'vscode-extension-tester';
+import { KNativeConstants } from './common/contants';
```

Comment:
`contants` has a typo and missing an S, should be `constants`, but the file name should be fixed too.